SECURITY.md

```diff
@@ -13,7 +13,10 @@ issue, and any other information that could aid an investigation. Our
 policy is to respond to vulnerability reports within 14 days.
 
 Our policy is to address critical security vulnerabilities rapidly and
-post patches as quickly as possible.
+post patches as quickly as possible.  If you do not get a response to
+a message sent to security@openexr.com within 48 hours, contact the
+project maintainers via a GitHub
+[Issue](https://github.com/AcademySoftwareFoundation/openexr/issues).
```

Comment:
Could I suggest https://github.com/AcademySoftwareFoundation/openexr/security/advisories/new instead of issue if you don't want the issues to be public? See https://docs.github.com/en/enterprise-cloud@latest/code-security/security-advisories/working-with-repository-security-advisories/about-repository-security-advisories for more info. Unless you want potential security issues to be disclosed right away, in which case github issues would work.

Reply:
That makes a lot of sense to me.

Reply:
Good suggestion, I added GitHub security advisories as the preferred method for reporting vulnerabilities, with email as an alternative, but also mentioned that if you don't get a response, your address may be blocked (it's happened).

Reply:
It looks great!